src/EditorFeatures/CSharpTest/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -1012,45 +1022,43 @@ public async Task TestDynamicIndexerAccess()
 @"class MyClass
 {
     private int[] _list;
-    private int [|this|][int index] => _list[index];
+    private int this[int index] => _list[index];
     public int M2(dynamic d) => this[d];
 }");
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
         public async Task FieldInDocComment()
         {
-            await TestDiagnosticsAsync(
+            await TestDiagnosticMissingAsync(
 @"
 /// <summary>
 /// <see cref=""C._goo""/>
 /// </summary>
 class C
 {
-    private static int [|_goo|];
-}",
-    expected: Diagnostic("IDE0052"));
+    private static int {|IDE0052:_goo|};
+}");
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
         public async Task FieldInDocComment_02()
         {
-            await TestDiagnosticsAsync(
+            await TestDiagnosticMissingAsync(
```

Comment:
This feels a bit weird as there is indeed a diagnostic (which seems to be validated from annotation), but just no code fix. Can we instead invoke `VerifyCS.VerifyCodeFixAsync(initialMarkup, initialMarkup)` directly here?

Reply:
➡️ Covered by https://github.com/dotnet/roslyn/pull/41755#discussion_r381076580